workers/merkle-tree-worker/src/worker.ts

```diff
@@ -0,0 +1,26 @@
+import { expose } from "comlink";
+import { IMerkleTreeWasm, IMerkleTreeWorker } from "./interface";
+
+let merkleTreeWasm: IMerkleTreeWasm;
+
+export const merkleTreeWorker: IMerkleTreeWorker = {
+    async prepare() {
```

Comment:
this does not comply with the `IMerkleTreeWorker` interface you defined.
does prepare return `void` or `Promise<void>`?